src/omega_edit/client.ts

```diff
@@ -69,8 +94,13 @@ export function activate(ctx: vscode.ExtensionContext) {
   ctx.subscriptions.push(
     vscode.commands.registerCommand(
       'omega_edit.version',
-      async (startServer: boolean = true) => {
-        await commonOmegaEdit(ctx, startServer, omegaEditPackageVersion)
+      async (
+        startServer: boolean = true,
+        port: number | undefined = undefined
+      ) => {
+        port = await getOmegaEditPort(port)
+        initOmegaEditClient('127.0.0.1', port.toString())
+        await commonOmegaEdit(ctx, startServer, omegaEditPackageVersion, port)
         return await omegaEditVersion.getVersion()
```

Comment:
This is invisible to the end-user using the GUI.  Should put the version in an info message as well.